MeetingNotes/2020-09-16-EveningCatchUp.md

```diff
@@ -0,0 +1,39 @@
+# Meeting Notes
+**Date:** Wednesday, 16th September 2020 - 7:30PM
+
+**Attendees:** Ana, Ellie, Jamie and Jyothsna
+
+**Topic:** Team Catch-up!
+
+## Topics:
+
+**1. Standard Meetings / Times that work for everyone**
+- The team agreed to catch up daily at 9am for 5-10 minutes, Monday to Friday, for a daily standup.
+- We also agreed to have a longer session on Wednesday's (9-9:30am) as a more in-depth team sync.
+
+**2. Definition of Done**
+- Jyothsna brought up discussing the definition of done. We explored a number of ideas and why we need a definition of done, and agreed:
+    * All code will be submitted as pull requests and peer reviewed by the team.
+    * We'll write tests for all work we complete. Tests written
+    * We'll write stories with clear requirements, and ensure the work completed meets these requirements.
+
+**3. Code Standards**
+- Jyothsna also raised setting some code standards, and we agreed:
+    * Using the “If applied, this commit will…” standard for commit messages (e.g.) "Change the background colour", "Fix a typo in the demo() method"
+    * Use PascalCase for filenames, repos and otherwhere appropriate
+
+**4. Methodology, Tools and Process*
+- We discussed what method to use and decided to use Agile, working in sprints with set sprint goals that we'll agree the Wednesday before sprint reviews.
+- We agreed to use Github Issues as our issue tracker tool.
+- We also agreed to store all team files on Github - a repo has been created ([TeamRepository](https://github.com/TheSentimentalists/TeamRepository)).
+
+**5. Wireframes/Solution**
+- We briefly talked about the various wireframes we we'd each created and agreed to submit Ellie's digital wireframes to Google Classroom.
+- Ellie agreed to finish these off and create a PDF we can submit.
+- We agreed to discuss and iterate on the designs further in the next sprint.
+- Ana brought up a number of libraries/backend ideas she has been exploring, including web scraping, APIs that have been looked at, platforms and libraries that could be used for the various processing we need to do.
+- We need to explore more whether we wish to focus on one topic (e.g. political news) or provide a general solution.
+- We need to explore what language to use on the backend - current suggestions are Java or Python.
+
+## Actions
+* The team to do some mob learning about Github, pull requests, and working collaboratively (a 'dry run').
```

Comment:
Actions I have: Demo early next week on 1) exploring project board for an issue tracker 2) plan for a  by Wednesday dry run on collaboration